very easy/19.js

```diff
@@ -1,12 +1,14 @@
 function sum(n) {
-	
+  let a = 0;

```

Comment:
`a` is the name of variable which say nothing to me

Reply:
name changed